### DISCUSSION TITLE
fail2ban config for webmail

### DISCUSSION THREAD

**Discussion by cayetona** [modoboa/modoboa#1659] [Q&A] [answered]
hi everyone, i need help to configure fail2ban with modoboa to prevent brut-force with the web interface.

I activate the log in settings.py  

        'syslog-auth': {
            'class': 'logging.handlers.SysLogHandler',
            'facility': SysLogHandler.LOG_AUTH,
            'address': '/dev/log',
            'formatter': 'syslog'
        },

i configure a jail.local

[modoboa]
enabled = true
port = http,https
protocol = tcp
filter = modoboa
maxretry = 3
bantime = 3600
logpath = /var/log/auth.log

but i have a problem to configure my filter ( /etc/fail2ban/filter.d/modoboa.conf

espacialy with the regex :

[INCLUDES]
before = common.conf
[Definition]
failregex = ip=<HOST>
ignoreregex =

the log i want to match with failregex look like this ::

Jan 19 00:25:01 mail modoboa.auth: WARNING Failed connection attempt from '0.0.0.0' as user 'user'

if anyone can help me.

Thx.

**Comment by tonioo** [comment]
failregex should contain something like this:
```ini
modoboa\.auth: WARNING Failed connection attempt from '<HOST>'
```

**Comment by Rhiannon81** [comment]
Hi, my fiter.d/modoboa.conf looks like this. Just the two lines, everything else is commented out.
```
[Definition]
failregex = modoboa.auth: WARNING Failed connection attempt from '<HOST>' as user '.+?'$
```

**Comment by JackApple1337** [comment, marked as answer]
I've fixed the `[...]/filter.d/modoboa.conf` for the newest version with Debian Stretch. Now it's working fine.
My full `modoboa.conf`:
```
[INCLUDES]  
before = common.conf
[Definition]
failregex = modoboa\.auth: WARNING Failed connection attempt from \'<HOST>\' as user \'.*?\'$
```

In addition I had to activate the UDP socket for rsyslogd in `/etc/rsyslog.conf`.
